src/utils/format/DateHelper.ts

```diff
@@ -1,6 +1,14 @@
 export const timestampToString = (timestamp: number) => {
+
+  if (!timestamp || isNaN(timestamp)) {
```

Comment:
oups, j'ai oublié ici 😅

```suggestion
  if (!timestamp || Number.isNaN(timestamp)) {
```